build-system/tasks/visual-diff/index.js

```diff
@@ -161,7 +161,7 @@ async function launchBrowser() {
   const browserOptions = {
     args: ['--no-sandbox', '--disable-extensions', '--disable-gpu'],
     dumpio: argv.chrome_debug,
-    headless: true,
+    headless: !argv.headless_disabled,
```

Comment:
We used to have a flag to set headless mode, but it was removed in #18510. Unfortunately, the tests don't work well in non-headless mode, because each tab has a different size, which can only be done in headless mode (otherwise each tab resizes the window, not just itself, and therefore resizes all the tabs)

I'm not sure this can be solved easily...